Rakefile

```diff
@@ -135,15 +135,15 @@ task documentation_syntax_check: :yard_for_generate_documentation do
 
     examples.each do |example|
       begin
-        # Parser output syntax errors to stderr, but it is not needed.
-        $stderr = StringIO.new(''.dup)
-        Parser::Ruby24.parse(example.text)
```

Comment:
https://github.com/whitequark/parser/blob/master/lib/parser/base.rb#L89
`Parser::Base.default_parser` outputs errors to STDERR, but the task no longer outputs by this change.